lib/std/os/uefi/protocols/absolute_pointer_protocol.zig

```diff
@@ -31,27 +31,29 @@ pub const AbsolutePointerProtocol = extern struct {
     };
 };
 
+pub const AbsolutePointerModeAttributes = packed struct(u32) {
+    supports_alt_active: bool,
+    supports_pressure_as_z: bool,
+};
```

Comment:
This won't compile, padding must be explicit even with the new semantics. The difference is that the compiler will check that the total size/alignment matches that of a u32

Reply:
Yeah, I just now realized this, fixing it as we speak